app/controllers/admin/calendar_events_controller.rb

```diff
@@ -8,6 +8,16 @@ def index
       project = Project.find(params[:project_id])
       authorize project, :show?
       project_filter = {id: project.id}
+    elsif params[:person_id]
+      person = Person.find(params[:person_id])
+      authorize person
+      authorize Project
+
+      project_ids = []
+      person.agent_projects.each do |project|
+        project_ids.push(project.id)
+      end
+      project_filter = {id: project_ids}
```

Comment:
Overall, though, nice job on incorporating this new functionality into the controller!